src/pkg/backup/details/iteminfo.go

```diff
@@ -36,6 +36,9 @@ const (
 
 	// Folder Management(30x)
 	FolderItem ItemType = 306
+
+	// GroupChannel(40x)
+	GroupChannel ItemType = 407
```

Comment:
Sanity check: if we go the route of treating channels as discrete items, then we should not plan to use them in the same way that we use Drives in onedrive and sharepoint.  If you'd prefer to handle channels the same way we handle drives, then we probably don't want a GroupChannel item type, in the same way that we don't have a Drive type.

Reply:
Agreed. It should be of `chatMessage`  type in that case. Changing it. Let me know if that fits best.